### PR TITLE
Adding PR template

### PR DESCRIPTION
**Pull request by mdazam1942**

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+**Solution Descrition:**
+
+A clear and concise solution description of the problem.
```
Comment (delliott90) [inline pull_request_template.md:3]: May want to reword as "A clear description of the problem and solution".

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+**Current Behavior:**
+
+Describe with example of the current behavior of the proposed code changes.
+
+**Unittest Added?** Yes/No
```
Comment (delliott90) [inline pull_request_template.md:9]: Probably change to "Unit tests". I like how you are getting devs to explain why they don't have any tests 😄

Comment (mdazam1942) [reply pull_request_template.md:9]: lol yes

File: pull_request_template.md (hunk 1 above)
Comment (mdazam1942) [reply pull_request_template.md]: sure

**Pull request merged**